browser/ui/webui/new_tab_page/brave_new_tab_page_handler.cc

```diff
@@ -194,6 +194,16 @@ void BraveNewTabPageHandler::UseSolidColorBackground(const std::string& color) {
   DeleteSanitizedImageFile();
 }
 
+void BraveNewTabPageHandler::UseGradientColorBackground(
+      const std::string& gradient_value) {
+  auto background_pref = NTPBackgroundPrefs(profile_->GetPrefs());
+  background_pref.SetType(NTPBackgroundPrefs::Type::kGradientColor);
+  background_pref.SetSelectedValue(gradient_value);
+
+  OnCustomBackgroundImageUpdated();
```

Comment:
nit: `OnCustomBackgroundUpdated()` would be more appropriate now because it's called when solid or gradient options are selected also.

Reply:
Good! Done.